### ISSUE TITLE
FR: Add an "Edit policy" button to snapshot view

### ISSUE THREAD

**Issue by nakermann1973** [kopia/kopia#3787]
When getting started using the UI (web ui via docker), I founf myself having to switch between policy and snapshot view fairly often to tune policies for specific snapshot users/paths.  having an "Edit Policy" button in the snapshot view would have made this simpler.

**Comment by nakermann1973** [comment]
I am using version 0.16.1.  My snapshot view does not have the Actions column

![image](https://github.com/kopia/kopia/assets/35577878/58524b41-fc21-4c33-9f03-49862deb8f8a)

**Comment by lupusA** [comment]
@nakermann1973 
Are you connected to the repo or to the kopia instance?
If nothing is shown, it means that the setHeader method defaulted. That would explain why no button appear at all. 

It should look like this:
![grafik](https://github.com/kopia/kopia/assets/37236531/207f756c-be02-4b5d-b520-c1596d3e3b62)


Cheers,

**Comment by nakermann1973** [comment]
Sorry for the delayed response.

I *do* see the Policy action button on the Kopia instance running on my local Windows PC which itself is connected to the docker repo server.

However, when I am directly connected to the WebUI of the docker repo (http://$IP:51515/snapshots), I see the snapshot view as in my [previous comment]( https://github.com/kopia/kopia/issues/3787#issuecomment-2044710958) with no action buttons.

I have upgraded to v0.17.0, with the same behaviour.

Which logs would be useful to help understand what is going on here?

**Comment by lupusA** [comment]
Hi @nakermann1973, 

then it is probably configured wrong. The above screenshot is from my docker container's web-interface. Have you taken a look at https://kopia.io/docs/installation/#docker-images ?

Cheers,

**Comment by nakermann1973** [comment]
> then it is probably configured wrong.   The above screenshot is from my docker container's web-interface.
It that for local snapshots (created by the container in the container) or remote snapshots which have been written into the repository from a client?  I have no local snapshots in my container, only remote hosts using the repository server.  I choose 'All snapshots' in the drop-down to view them.  My default view when loading the web ui is 
![image](https://github.com/kopia/kopia/assets/35577878/80481829-278f-4a5b-a9af-8208b4f29643)

>  Have you taken a look at https://kopia.io/docs/installation/#docker-images ?
I did. That's where I got my initial configuration from:

```
  kopia:
    image: kopia/kopia:latest
    restart: unless-stopped
    container_name: kopia
    hostname: kopia
    ports:
      - 51515:51515
    volumes: 
      - kopia_conf:/app/config
      - /srv/backup/kopia/repository:/repository
      - /srv/backup/kopia/logs:/app/logs
      - /srv/backup/kopia/cache:/app/cache
    environment:
      KOPIA_PASSWORD: MY_REPOSITORY_PASSWORD
      # The kopia server control address, username and password - for issuing server-related commands in the container
      KOPIA_SERVER_USERNAME: server-control # To control the kopia server
      KOPIA_SERVER_PASSWORD: MY_SERVER_PASSWORD # To control the kopia server
      KOPIA_SERVER_ADDRESS: https://127.0.0.1:51515
      KOPIA_SERVER_CERT_FINGERPRINT: MY_FINGERPRINT
    command: [
      "server",
      "start",
      "--address=0.0.0.0:51515",
      "--server-control-username=server-control", # For server control
      "--server-control-password=MY_SERVER_PASSWORD",  # For server control
      "--server-username=kopia_ui",  # For the UI
      "--server-password=MY_UI_PASSWORD",  # For the UI
      "--tls-cert-file=/app/config/tls/kopia.cert",
      "--tls-key-file=/app/config/tls/kopia.key",
```

What I am missing is the following volume options:

```           
           # Mount local folders to snapshot
            - /path/to/data/dir:/data:ro
            # Mount path for browsing mounted snaphots
            - /path/to/tmp/dir:/tmp:shared
```

Since I don't plan to use either, I left them out.  Those directories are still present in the container though, just empty.

**Comment by nakermann1973** [comment]
I have just tested my theory that the buttons only show up for local snapshots.  I created a snapshot using the 'New snapshot' button in the web ui, and for this snapshot, the action buttons are present, but not for the snapshots written from a remote kopia instance:

![image](https://github.com/kopia/kopia/assets/35577878/ba12ebbb-52eb-4682-98b0-14674ee118f9)

This makes me that this is a bug rather than a feature request.

**Comment by lupusA** [comment]
Hi @nakermann1973, 

so the request is, that you would like to see the "edit" button for remote clients, when running a repository server?

Cheers

**Comment by nakermann1973** [comment]
@lupusA yes, that sums it up.  When originally logging this I had not realized that this only affected those running a repository server with no local snapshots.
